test/functional/operation_array_update_example_test.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+const setupDatabase = require('./shared').setupDatabase;
+const expect = require('chai').expect;
+
+describe('Array Filter Update Example', function() {
+  before(function() {
+    return setupDatabase(this.configuration);
+  });
+
+  it('supports using array filters when updating one', {
+    metadata: {
+      requires: {
+        mongodb: '>=3.6.x',
+        topology: ['single']
+      }
+    },
+
+    test: function(done) {
+      const configuration = this.configuration;
+      const client = configuration.newClient(configuration.writeConcernMax(), { poolSize: 1 });
+      client.connect(function(err, client) {
+        const db = client.db(configuration.db);
+        const collection = db.collection('arrayFilterUpdateExample');
+
+        // 3. Exploiting the power of arrays
```

Comment:
also does this line need a corresponding `end` tag?

Reply:
I will check with the docs team 👍